src/utils/dataFormatters/dataFormatters.ts

```diff
@@ -63,13 +68,28 @@ export const formatStorageValues = (value?: number, total?: number, size?: Bytes
         valuePrecision = 1;
     }
 
-    const formattedValue = formatBytesCustom({
+    let formattedValue = formatBytesCustom({
         value,
         withSizeLabel: valueWithSizeLabel,
         size: size || calculatedSize,
         precision: valuePrecision,
     });
-    const formattedTotal = formatBytesCustom({value: total, size: size || calculatedSize});
+    if (value && value > 0) {
+        while (formattedValue === '0') {
+            valuePrecision += 1;
```

Comment:
please add some condition to prevent infinite loop here

Reply:
It seems no infinite loops is available here, because before enter the cycle I check if `value` is not nullish.